include/ace/managers/viewport/tilebuffer.h

```diff
@@ -199,7 +199,15 @@ void tileBufferReset(
  * Be sure to have display turned off or palette dimmed since even on double
  * buffering it will redraw both buffers.
  */
-void tileBufferRedrawAll(tTileBufferManager *pManager);
+void tileBufferRedrawAllNonInterleaved(tTileBufferManager *pManager);
```

Comment:
If I read this correctly, then all functions are fine to call at any time because they don't require extra flags to work, besides interleaved.

The doc comment needs some info what are the tradeoffs between them so that user can make an informed decision.

The typical ACE way is to either pass a tag to create that switches the behavior, or setting a define that rewires the code. I'm not sure how I feel about that third way, especially that user will probably only go with one of those two ways exclusively on a per-project basis. I need some more time/opinions on this to tell if it's ok or not.

Reply:
Sure! I guess we can also take the performance hit from the `if (ubSrcInterleaved && ubDstInterleaved) {` as it's probably negligible in total frame time. Or if it proves problematic sometime we can cache it in a variable or something.

Reply:
Pushed with the small amount of duplicated code.